src/commands/help.js

```diff
@@ -113,26 +211,20 @@ path to .http file name or url
 
 -e, --environment
 
-  Environment file path.
-
-  Environment file a json file with variables and their values (similar to -i parameter) it allows you to reuse existing .http files.
-  For instance you can have following sample.http file:
-
-  POST https://@@apiUrl@@/value
```

Comment:
Please add this in as example when describing environments. Ideally, show config file with 2 envs and apiUrl input variable